packages/lib/models/Note.test.ts

```diff
@@ -332,4 +332,39 @@ describe('models_Note', function() {
 		expect(sortedNotes3[4].id).toBe(note2.id);
 	}));
 
+	it('should copy the note to target folder and cancel conflict', (async () => {
+		const folder1 = await Folder.save({ title: 'Folder' });
+		const conflictFolder = await Folder.save({ title: Folder.conflictFolderTitle(), id: Folder.conflictFolderId() });
```

Comment:
The conflict folder is a virtual one so please don't create it. Have a look at the [code here](https://github.com/laurent22/joplin/pull/5049/files#diff-6cc42976df77c04be963c9986bb17b91dc8aa4bc12743b34f5004a7c1d0e3692R613-R617) which shows how a conflict note is created. It's the exact same as the original note but with is_conflict set to 1 and a different ID. Its parent is not the conflict folder. The conflict folder is just GUI stuff that you don't need to worry about at the model level.